terraform-sentinel/index.json

```diff
@@ -0,0 +1,48 @@
+{
+  "title": "Sentinel Tutorial",
+  "description": "Writing & Testing Sentinel Policies in the Sentinel CLI and publishing to TFC",
+  "time": "30 minutes",
+  "icon": "fa-terraform",
+  "details": {
+    "intro": {
+        "text": "intro.md",
+        "code": "courseBase.sh",
+        "credits": "https://learn.hashicorp.com/terraform"
+    },
+        "assets": {
+      "host01": [
+
+        { "file": "restrict-s3-buckets.sentinel", "target": "~/" }
+      ]
+    },
+    "steps": [
+        {
+          "title": "Your first Sentinel Policy",
```

Comment:
Maybe more declarative: "Write your first Sentinel policy"?
Also, should it be a lowercase "policy"?